packages/create/README.md

```
@@ -26,6 +26,26 @@ pnpm dlx @gwigz/slua-create my-project
 bunx @gwigz/slua-create my-project
 ```
 
+### Options
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)